lib/base/networking/protocols/grpc/google/api/annotations.pb.dart

```diff
@@ -0,0 +1,27 @@
+//
```

Comment:
maybe move the google folder to protocols?

Reply:
Because of import reasons, this needs to be a subfolder in the folder structure `google/api/...`
Otherwise, regenerating this would require extra manual work

Reply:
I have moved them into the api folder as the singleton removes the need for a service